x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/outlier_exploration/outlier_exploration.tsx

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+
+import { i18n } from '@kbn/i18n';
+
+import {
+  EuiCallOut,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiPanel,
+  EuiSpacer,
+  EuiTitle,
+} from '@elastic/eui';
+
+import {
+  useColorRange,
+  ColorRangeLegend,
+  COLOR_RANGE,
+  COLOR_RANGE_SCALE,
+} from '../../../../../components/color_range_legend';
+
+import { sortColumns, INDEX_STATUS, defaultSearchQuery } from '../../../../common';
+
+import { DATA_FRAME_TASK_STATE } from '../../../analytics_management/components/analytics_list/common';
+import { getTaskStateBadge } from '../../../analytics_management/components/analytics_list/columns';
+
+import { useExploreData, TableItem } from '../../hooks/use_explore_data';
+
+import { ExplorationDataGrid } from '../exploration_data_grid';
+import { ExplorationQueryBar } from '../exploration_query_bar';
+
+const FEATURE_INFLUENCE = 'feature_influence';
```

Comment:
This is also defined in `exploration_data_grid.tsx` - perhaps this could be moved to a shared location and imported into both places it's used?